<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GParseWidget</class>
 <widget class="QWidget" name="GParseWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>1051</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0,0,0,0,0,0">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="checkBoxQP">
         <property name="text">
          <string>Lekérdezés fordító</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labelQPType">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Típus :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxQP">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonMap">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Paraméterek</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="labelQP">
       <property name="text">
        <string>Exportált, lefordítandó szöveg :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEditQP"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutQP">
       <item>
        <widget class="QPushButton" name="pushButtonLoadQP">
         <property name="text">
          <string>Beolvas</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSaveQP">
         <property name="text">
          <string>Elment</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonClearQP">
         <property name="text">
          <string>Töröl</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonParseQP">
         <property name="text">
          <string>Végrehajt</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labelSrc">
         <property name="text">
          <string>Forrás szöveg:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="textEditSrc">
       <property name="acceptRichText">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="pushButtonLoad">
         <property name="text">
          <string>Beolvas</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSave">
         <property name="text">
          <string>Elment</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonClear">
         <property name="text">
          <string>Töröl</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonBreak">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Megszakít</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonParse">
         <property name="text">
          <string>Végrehajt</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>A végrehajtás helye :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonLocal">
       <property name="text">
        <string>Helyben</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonServer">
       <property name="text">
        <string>Szerveren</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="labelResult">
           <property name="text">
            <string>Eredmény:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="textEditResult">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="undoRedoEnabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="labelLog">
           <property name="text">
            <string>Hiba csatorna, napló :</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="textEditLog">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="undoRedoEnabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="text">
        <string>Becsuk</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>radioButtonServer</zorder>
  <zorder>radioButtonLocal</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonClear</sender>
   <signal>clicked()</signal>
   <receiver>textEditResult</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>537</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonClear</sender>
   <signal>clicked()</signal>
   <receiver>textEditSrc</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonClearQP</sender>
   <signal>clicked()</signal>
   <receiver>textEditQP</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>345</y>
    </hint>
    <hint type="destinationlabel">
     <x>649</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxQP</sender>
   <signal>toggled(bool)</signal>
   <receiver>textEditQP</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>649</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxQP</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBoxQP</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>1228</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxQP</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButtonClearQP</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxQP</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButtonLoadQP</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxQP</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButtonParseQP</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>1231</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxQP</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButtonSaveQP</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxQP</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelQP</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>649</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxQP</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelQPType</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>1130</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxQP</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButtonMap</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>1214</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
